TPHealth/TPHealth/ThirdPartyNotices.txt

```diff
@@ -2,7 +2,7 @@
 THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
 Do Not Translate or Localize
 
-Team Project Health incorporates third party material from the projects listed below. The original copyright notice and the license under which Microsoft received such third party material are set forth below. Microsoft reserves all other rights not expressly granted, whether by implication, estoppel or otherwise. 
+Visualize-Team-Project-Health-WidgetsALM | DevOps Ranger Contributors incorporates third party material from the projects listed below. The original copyright notice and the license under which ALM | DevOps Ranger Contributors received such third party material are set forth below. ALM | DevOps Ranger Contributors reserve all other rights not expressly granted, whether by implication, estoppel or otherwise. 
```

Comment:
Visualize-Team-Project-Health-WidgetsALM | DevOps Ranger Contributors incorporates third ... should read:
Visualize-Team-Project-Health-Widgets incorporates third ...

There seems to be a random ALM | DevOps Rangers paste.